src/Extension/MemberExtension.php

```diff
@@ -31,95 +35,31 @@ class MemberExtension extends Extension implements PermissionProvider
     {
-        $fields->removeByName(['LoginSessions', 'SessionManagerField']);
+        $fields->removeByName('LoginSessions');
 
-        if (!$this->owner->exists() || !$this->currentUserCanViewSessions()) {
-            return $fields;
+        $member = $this->getOwner();
+        if (!$member->exists()) {
+            // We're creating a new member
+            return;
         }
 
+        $session = $member->LoginSessions()->First();
+        if (!$session || !$session->canView()) {
+            // We're assuming that the first session permission are representative of the other sessions
+            return;
+        }
+
+        $helpUrl = $member::config()->get('session_login_help_url');
+
         $fields->addFieldToTab(
             'Root.Main',
-            $sessionManagerField = SessionManagerField::create(
-                'SessionManagerField',
+            SessionManagerField::create(
+                'LoginSessions',
                 _t(__CLASS__ . '.DEVICES', 'Authenticated devices'),
                 $this->owner->ID,
-                _t(__CLASS__ . '.LEARN_MORE', 'Learn more'),
-                'http://userhelp.silverstripe.org/TODO'
-            )
-        );
-
-        if (!$this->currentUserCanEditSessions()) {
-            $sessionManagerField->setReadonly(true);
-        }
-
-        return $fields;
-    }
-
-    /**
-     * @return int
-     */
-    private function getSessionLifetime(): int
-    {
-        if ($lifetime = Session::config()->get('timeout')) {
-            return $lifetime;
-        }
-
-        return LoginSession::config()->get('default_session_lifetime');
-    }
-
-    /**
-     * Determines whether the logged in user has sufficient permission to see the SessionManager config for this Member.
-     *
-     * @return bool
-     */
-    public function currentUserCanViewSessions(): bool
-    {
-        return (Permission::check(self::SESSION_MANAGER_ADMINISTER_SESSIONS)
-            || $this->currentUserCanEditSessions());
-    }
-
-    /**
-     * Determines whether the logged in user has sufficient permission
-     * to modify the SessionManager config for this Member.
-     * Note that this is different from being able to _reset_ the config (which administrators can do).
-     *
-     * @return bool
-     */
-    public function currentUserCanEditSessions(): bool
-    {
-        return (Security::getCurrentUser() && Security::getCurrentUser()->ID === $this->owner->ID);
-    }
-
-    /**
-     * Provides the SessionManager view/reset permission for selection in the permission list in the CMS.
-     *
-     * @return array
-     */
-    public function providePermissions(): array
-    {
-        $label = _t(
-            __CLASS__ . '.SESSION_MANAGER_PERMISSION_LABEL',
-            'View/purge login sessions for other members'
+                $helpUrl ? _t(__CLASS__ . '.LEARN_MORE', 'Learn more') : '',
+                $helpUrl ?: ''
+            )->setReadonly(!$session->canDelete())
```

Comment:
We don't currently have a read only view for the field https://github.com/silverstripe/silverstripe-session-manager/issues/63

Reply:
What happens on the frontend if readonly is set?  Does it just fallback to the read-view?

Reply:
Did try it, but my guess is that it will show you a logout button and you'll get an error if you try clicking it.

Out of the box, it's not possible to get to that state right now.

Reply:
If we do not support properly support it, and you cannot get this state (canView is true and canDelete is false) then we should just remove this code.  It is confusing right now.

If we add in future support for admins viewing other peoples login sessions, then we should add this line back in.  It's pretty easy to do so.  At that point we can also actually prioritise building a readonly view of the component.

Reply:
My hope is that we will eventually implement #63 and have some level of support for it. 

Since we're writing doc to explain to people how to update their Login permission, sounds like we expect at least some people to have an interest in this scenario.

Reply:
Noting that this was called out as a task in the initial plan: https://github.com/silverstripe/silverstripe-session-manager/issues/57#issuecomment-819160150:

>Set SessionManagerField to readonly if the user doesn't have canDelete permission and spin off an issue to implement a read-only view for this field.